HACKING.md

```diff
@@ -106,6 +106,46 @@ otherwise an official image can be run with
 
 FAF should be available at ```http://localhost:8080/faf/```
 
+### Reporting into FAF
+1. Set up your server URL for libreport
+
+    Insert `URL = http://localhost:8080/faf` into `/etc/libreport/plugins/ureport.conf`
+2. Create crash
+
+    Install package `will-crash` and then execute `will_abort`
+
+This should be enough, so in a few minutes (reports are processed every few minutes) you should see
+spike in your FAF. However there are a few places where it can go wrong, therefore following
+troubleshooting:
+
+
+
+1. Was the problem caught by ABRT on your local machine?
+
+    Check if you have something in `/var/spool/abrt` (mind you, must read as root, otherwise may look empty even if is not empty). There should be some subdirectory starting with "cccp". If you dont have there anything like that, probably ABRT is not running on your machine. Try reading https://abrt.readthedocs.io/en/latest/ .
```

Comment:
You can check if the crash was caught by abrt with just `abrt-cli ls` (or if you have `abrt-cli-ng` package `abrt ls`) and it will list all the crashes of the user.

Reply:
Thats a good point :) I just use ABRT in a strange way, but its better to use it this way - will change it.